eng/common/scripts/Create-APIReview.ps1

```diff
@@ -82,17 +83,19 @@ else
 }
 
 $FoundFailure = $False
-$pkgInfoPath = Join-Path -Path $ArtifactPath "PackageInfo"
+if (-not $ConfigFileDir)
+{
+    $ConfigFileDir = Join-Path -Path $ArtifactPath "PackageInfo"
+}
 foreach ($pkgName in $responses.Keys)
 {    
     $respCode = $responses[$pkgName]
     if ($respCode -ne '200')
     {
-        $pkgPropPath = Join-Path -Path $pkgInfoPath ($PackageName + ".json")
+        $pkgPropPath = Join-Path -Path $ConfigFileDir ($PackageName + ".json")
```

Comment:
nit: would `Join-Path -Path $ConfigFileDir "$PackageName.json"` read easier?

Reply:
Thanks for this suggestion.